src/terms/local.jl

```diff
@@ -104,26 +117,30 @@ function compute_local_potential(basis::PlaneWaveBasis{T}; positions=basis.model
     # and a structure factor e^{-i G·r}
     model = basis.model
     Gqs_cart = [model.recip_lattice * (G + q) for G in to_cpu(G_vectors(basis))]
```

Comment:
If this is no longer needed in the rest of the function this should be moved into `atomic_local_form_factors` in my opinion.